src/pagerprinter/plugins/email.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+"""
+SMTP plug-in
+Copyright 2014 Shane Rees
+
+This allows call out to be sent via email 
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>
+
+This plug-in send the page as an email. You must be aware that
+there can be a delay in receiving the email/page.
+
+"""
+from __future__ import absolute_import
+from . import BasePlugin
+import smtplib
+from email.mime.text import MIMEText
+
+
+
+# make our plugin!
+class email(BasePlugin):
+    def configure(self, c):
+        # username
+        self.username = [
+            x.strip()
+            for x
+            in c.get('email', 'username').split(',')
+        ]
+    def configure(self, c):
+        # password
+        self.password = [
+            x.strip()
+            for x
+            in c.get('email', 'password').split(',')
+        ]		
+
+    def configure(self, c):
+        # read in emails we need
+        self.recipient = [
+            x.strip()
+            for x
+            in c.get('email', 'to').split(',')
+        ]
+    def configure(self, c):
+        # who its from
+        self.sender = [
+            x.strip()
+            for x
+            in c.get('email', 'from').split(',')
+        ]	
+    def configure(self, c):
+        # read in emails we need
+        self.server2 = [
+            x.strip()
+            for x
+            in c.get('email', 'server').split(',')
+        ]
+    def execute(self, msg, unit, address, when, printer, print_copies):
+        msg = MIMEText ('%s - %s' % (msg, unit))
+        
+        for recipient in self.recipient:
```

Comment:
To avoid confusion, for `self.recipients`, you should maybe use the plural form (eg: `self.recipients`) instead.  You now have a local variable called `recipient` and an instance variable called `recipient`.